src/plugins/git/address.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import type {Address} from "../../core/address";
+import type {EdgeType, NodeType} from "./types";
+import {COMMIT_NODE_TYPE, GIT_PLUGIN_NAME} from "./types";
+
+export function makeAddress(type: NodeType | EdgeType, id: string): Address {
```

Comment:
As discussed, this should be package-private: it enables clients who only call the public API to get invalid addresses (by supplying invalid IDs). But JavaScript doesn't have a package-private visibility option. Please either (a) underscore-name this, or (b) export all the individual `fooAddress` functions instead of this.